crates/cxx-qt-gen/update_expected.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+
+set -ex
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
```

Comment:
shellcheck warns for the following (basically add quotes)

```
In crates/cxx-qt-gen/update_expected.sh line 12:
cd ${SCRIPT_DIR}
   ^-----------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
cd "${SCRIPT_DIR}"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```